docs/reference/commandline/ps.md

```diff
@@ -174,9 +174,19 @@ Any of these events result in a `137` status:
 
 #### status
 
-The `status` filter matches containers by status. You can filter using
-`created`, `restarting`, `running`, `removing`, `paused`, `exited` and `dead`. For example,
-to filter for `running` containers:
+The `status` filter matches containers by status. The possible values for the container status are:
+
+| Status                | Description                                                                                                                          |
+|:----------------------|:-------------------------------------------------------------------------------------------------------------------------------------|
+| `created`             | A container that has never been started. |
+| `running`             | A running container, started by either `docker start` or `docker run`. |
```

Comment:
I was curious if the filter would show paused (but running) containers as "running", but it looks like we don't show paused containers when filtering on `running`

(We probably have some wording somewhere though to explain that `paused` and `running` are not mutually exclusive)

Reply:
Ok, I won't worry about adding this right now.